drivers/docker/driver.go

```diff
@@ -835,6 +835,12 @@ func (d *Driver) createContainerConfig(task *drivers.TaskConfig, driverConfig *T
 		return c, fmt.Errorf("requested runtime %q is not allowed", containerRuntime)
 	}
 
+	// Only windows supports alternative isolations modes
+	isolationMode := driverConfig.Isolation
+	if runtime.GOOS != "windows" && driverConfig.Isolation != "process" {
```

Comment:
It doesn't look like the configuration sets `"process"` as the default, so I think it's `""`, right? Won't that cause us to hit this error for every non-Windows OS?

Reply:
Yes, can confirm now from testing.